### PR TITLE
Update 03-nareshpulipati.md

### PR DESCRIPTION
**Pull request by nareshpulipati**
test

### CODE REVIEW

File: _slides/03-nareshpulipati.md [hunk 1]
```diff
@@ -3,7 +3,7 @@ layout: slide
 title: "Welcome to our slide deck!"
 ---
 
-Caption here
+New caption
```
Comment (prabhakar-gundu) [inline _slides/03-nareshpulipati.md]: Can you put your name here

**Review by prabhakar-gundu — CHANGES_REQUESTED** [review]
Added a review point

File: _slides/03-nareshpulipati.md (hunk 1 above)
Comment (nareshpulipati) [reply _slides/03-nareshpulipati.md]: @prabhakar-gundu  added

**Review by prabhakar-gundu — APPROVED** [review]
Changes are good

**Pull request merged**